.github/workflows/test.yaml

```diff
@@ -138,8 +173,8 @@ jobs:
         run: |
           make docker-build-eraser ERASER_IMG=${{ env.REGISTRY }}/${REPO}:test
 
-      - name: Run trivy for eraser 
+      - name: Run trivy for eraser
         run: trivy image --ignore-unfixed --exit-code=1 --vuln-type=os,library ${{ env.REGISTRY }}/${REPO}:test
-          
-      - name: Run trivy for eraser-manager 
-        run: trivy image --ignore-unfixed --exit-code=1 --vuln-type=os,library ${{ env.REGISTRY }}/${REPO}-manager:test
\ No newline at end of file
+
+      - name: Run trivy for eraser-manager
+        run: trivy image --ignore-unfixed --exit-code=1 --vuln-type=os,library ${{ env.REGISTRY }}/${REPO}-manager:test
```

Comment:
opened https://github.com/Azure/eraser/issues/233 so we can add collector and scanner here too